json/test_rapid.cpp

```diff
@@ -38,7 +38,7 @@ string read_file(const string& filename) {
 
 coordinate_t calc(const string& text) {
   Document jobj;
-  jobj.Parse(text);
+  jobj.Parse<kParseFullPrecisionFlag>(text);
```

Comment:
Do you know does it affect performance? If yes (i.e. the time diff > 10%), we may have 2 tests based on that code.

Reply:
I didn't test, unlikely for the current file it will be a changer. However, for the new ruby generation script, it may be.